packages/web/src/components/comments/CommentForm.tsx

```diff
@@ -10,19 +11,22 @@ import {
   LoadingSpinner
 } from '@audius/harmony'
 import { Form, Formik, useFormikContext } from 'formik'
+import { useSelector } from 'react-redux'
 import { usePrevious } from 'react-use'
 
 import { TextField } from 'components/form-fields'
 import { useProfilePicture } from 'hooks/useUserProfilePicture'
+import { audioPlayer } from 'services/audio-player'
 
 type CommentFormValues = {
   commentMessage: string
 }
 
 type CommentFormProps = {
-  onSubmit: (commentMessage: string) => void
+  handleSubmit?: ({ commentMessage }: { commentMessage: string }) => void
```

Comment:
We usually use `onSubmit` as the name for the prop and then `handleSubmit` is the function implementation in a component